subjects/devops/cloudPress/audit/README.md

```diff
@@ -0,0 +1,97 @@
+
+###### Were security groups, storage, and networking configurations set up correctly using Terraform?
+
+###### Was the EC2 Instance securely configured after setup (SSH, security, etc.)?
+
+#### Nginx Installation and Configuration using Ansible
+
+###### Was Nginx successfully installed on the EC2 Instance using Ansible?
+
+###### Was Nginx configured to serve web content appropriately?
+
+###### Is Nginx service running? Use `systemctl status nginx`.
+
+###### Were firewall settings adjusted to allow HTTP/HTTPS traffic?
+
+#### MariaDB Installation and Configuration using Ansible
+
+###### Was MariaDB installed on the EC2 Instance using Ansible?
+
+###### Is MariaDB service active? Use `systemctl status mariadb`.
+
+###### Was a MariaDB database created specifically for WordPress?
+
+###### Were the user rights and privileges for the WordPress user correctly configured?
+
+#### PHP Installation and Configuration using Ansible
+
+###### Was PHP installed on the EC2 Instance using Ansible?
+
+###### Is PHP-FPM running? Use `systemctl status php-fpm`.
+
+###### Were PHP settings appropriately configured for the WordPress site?
+
+###### Was the PHP service successfully started?
+
+#### WordPress Installation and Configuration using Ansible
+
+###### Was WordPress downloaded and set up on the EC2 Instance using Ansible?
+
+###### Was WordPress configured to use the MariaDB database?
+
+###### Is the database connectivity confirmed?
+
+###### Was the WordPress service successfully started?
+
+###### Using curl or browser: Is WordPress accessible?
+
+###### Can users access the WordPress site without encountering errors or downtime?
+
+###### Was the environment configuration secure and effective?
+
+Consider the following examples of secure and effective environment configuration:
+
+- **Sensitive Information Handling:** Ensure sensitive information like passwords, API keys, or access credentials are not hardcoded in configuration files. Utilize specialized tools such as AWS Secrets Manager, HashiCorp Vault, or equivalent solutions to securely manage and access sensitive data.
+
+- **Scalability Measures:** Check for implemented strategies to handle traffic spikes or increased loads, such as auto-scaling mechanisms or load balancing.
+
+- **Fault Tolerance:** Validate the setup's ability to manage various failure scenarios, maintaining high availability of the WordPress site.
+
+###### Are sensitive information such as passwords and access keys securely stored and managed (e.g., using AWS Secrets Manager or similar tools)?
+
+###### Were proper encryption methods implemented for data in transit and at rest?
+
+#### Documentation
+
+###### Does the README.md file contain an architecture overview?
+
+###### Does the documentation cover the deployment process comprehensively?
+
+###### Are additional configurations, tips, and potential pitfalls included in the documentation?
+
+#### Technical Verification
+
+##### Try to change Terraform Settings.
```

Comment:
I would change specify the question a bit. “Change some Terraform settings as instructed below:”